airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -117,13 +117,19 @@ def build_task_instance_run_id(dag_id, task_id, execution_date, try_number):
             )
         )
 
-    def emit(self, event: RunEvent):
+    def emit(self, event: RunEvent) -> RunEvent:
+        """Emit OpenLineage event.
+
+        :param event: Event to be emitted.
+        :return: Redacted Event.
+        """
         if not self._client:
             self._client = self.get_or_create_openlineage_client()
         redacted_event: RunEvent = self._redacter.redact(event, max_depth=20)  # type: ignore[assignment]
         try:
-            with Stats.timer("ol.emit.attempts"):
-                return self._client.emit(redacted_event)
+            with Stats.timer(f"ol.emit.attempts.{event.eventType.value}.{self._client.transport.kind}"):
```

Comment:
I'm not sure if we want to replace this metric or add a new one next to it?

Reply:
Looking at the statsd lack of tags, I would emit both